modules/evvtMenu/index.php

```diff
@@ -26,9 +26,11 @@
 $menu_structure = getMenuBranch(0);
 $elements = getMenuPicklist(0, 0);
 $json = getMenuJSON2();
+$getsavedM =getsavedMenu();
```

Comment:
I understand that you are sending all the saved menus to the browser. Is that necessary?